server/server/session/src/main/java/com/alipay/sofa/registry/server/session/push/PushEfficiencyImproveConfig.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.registry.server.session.push;
+
+import com.alipay.sofa.registry.server.session.bootstrap.SessionServerConfig;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.commons.collections.CollectionUtils;
+
+/**
+ * @author jiangcun.hlc@antfin.com
+ * @since 2023/3/16
+ */
+public class PushEfficiencyImproveConfig {
+
+  public static final int DEFAULT_CHANGE_TASK_WAITING_MILLIS = 100;
```

Comment:
all config should be changeable from springboot config, just like SessionServerConfig.

Reply:
These same wait time configurations were originally hard-coded, the Feature support dynamic configuration, so no need springboot config